src/modules/invite/invite.controller.ts

```diff
@@ -41,4 +44,29 @@ export class InviteController {
       }
     }
   }
+
+  @Post(':orgId/invite/:inviteId/refresh')
+  async refreshInvitation(@Param('orgId') orgId: string, @Param('inviteId') inviteId: string) {
+    if (isUUID(orgId, 4)) {
```

Comment:
Validation is best handled in the controllers. Service layer is for your bussiness logic